cypress/support/commands/action.js

```diff
@@ -1,4 +1,15 @@
 function executeHeaderAction(actionName) {
+  /*
+  if this is run straight after visiting a window, the action is not loaded, and everything gets broken.
+  example:
+
+    cy.visitWindow('540154);
```

Comment:
i think these 3 lines don'T add value and can go; correct?

Reply:
Not yet, as i havent fixed that. will do it in https://github.com/metasfresh/metasfresh-e2e/issues/130 hopefully

Reply:
NOT fixed!